etc/kayobe/ansible/requirements.yml

```diff
@@ -7,7 +7,7 @@ collections:
   - name: pulp.squeezer
     version: 0.0.13
   - name: stackhpc.pulp
-    version: 0.5.4
+    version: 0.5.5
```

Comment:
Maybe worth a release note? It's the sort of thing that's an easy win if we're planning on generating diffs for clients

Reply:
It doesn't provide any user-impacting changes, so I'm inclined to leave it this time. This did remind me to add a reno for https://github.com/stackhpc/stackhpc-kayobe-config/pull/937 though.